gene/query.py

```diff
@@ -160,79 +144,14 @@ def fill_no_matches(self, resp: Dict) -> Dict:
                     'records': [],
                     'source_meta_': self.fetch_meta(src_name)
                 }
+            else:
+                # sort records based on highest match_type
+                records = resp['source_matches'][src_name]['records']
+                if len(records) > 1:
+                    records = sorted(
+                        records, key=lambda k: k.match_type, reverse=True)
```

Comment:
IMO this should dictate a name change for this method ("post process query" or something?) , or be moved further up in the call stack